src/persistence/job.js

```diff
@@ -12,18 +12,58 @@ module.exports = {
       }, ${jobParameters.description}, current_date, ${jobParameters.tags}, ${
         jobParameters.company
       }, ${jobParameters.logoURL})
-        RETURNING id, title, tags;
+        RETURNING *;
       `);
       return rows[0];
     } catch (error) {
       throw error;
     }
   },
-
   async find(id) {
     const {rows} = await db.query(sql`
     SELECT * FROM jobs WHERE id=${id} LIMIT 1;
     `);
     return rows[0];
+  },
+  async update(isJobExist, jobParameters) {
+    try {
+      let newTitle = jobParameters.title;
+      let newSalaryRange = jobParameters.salaryRange;
+      let newDescription = jobParameters.description;
+      let newTags = jobParameters.tags;
+      let newCompany = jobParameters.company;
+      let newLogoURL = jobParameters.logoURL;
+      if (newTitle.trim() === '') {
```

Comment:
Should we setup `newTitle` as a constant in the beginning like this
```
const newTitle = jobParameters.title.trim();
```
?

Is it better?

Reply:
It's better to use trim() function right at the time we declare it, but I want it to be able to change in case the user let it null or blank so I can set it to the old information instead. I assume that those fields can not be blank.

Reply:
Ok I get what you mean. But is there any different between your code like this:
```
let newTitle = jobParameters.title;
...
if (newTitle.trim() === '') {
        newTitle = isJobExist.title;
      }
```

and the suggestion like this?
```
const newTitle = jobParameters.title.trim();
...
if (newTitle === '') {
newTitle = isJobExist.title;
}
```